test/test-contrast.js

```
@@ -0,0 +1,178 @@
+const fs = require('node:fs');
+const path = require('node:path');
+
+const THEME_DIR = path.join(__dirname, '..', 'themes');
+const MIN_CONTRAST_RATIO = 4.5; // WCAG AA standard for normal text
+
+// Convert hex to RGB
+function hexToRgb(hex) {
+  // Remove # if present
+  hex = hex.replace(/^#/, '');
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)